hcloud/image.go

```diff
@@ -92,6 +93,14 @@ func (c *ImageClient) GetByName(ctx context.Context, name string) (*Image, *Resp
 	return ImageFromSchema(body.Images[0]), resp, nil
 }
 
+// GetByIDOrName retreives an image by its id if the input can be parsed as int or by its name.
```

Comment:
“retreives” is misspelled and “id” should be uppercase.

I think it can be formulated more clearly: “GetByIDOrName retrieves an image by its ID if the input can be parsed as an integer, otherwise it retrieves an image by its name.”